client/components/global-notices/index.jsx

```diff
@@ -77,14 +77,16 @@ const NoticesList = React.createClass( {
 		//needs to be updated.
 		noticesList = noticesList.concat( this.props.storeNotices.map( function( notice, index ) {
 			return (
-				<Notice
-					key={ 'notice-' + index }
-					status={ notice.status }
-					duration = { notice.duration || null }
-					showDismiss={ notice.showDismiss }
-					onDismissClick={ this.props.removeNotice.bind( this, notice.noticeId ) }
-					text={ notice.text }>
-				</Notice>
+				<div className='global-notices__notice'>
```

Comment:
I think it could be removed. I no longer remember why it was there and after some quick testing, looks like verything still works without it.